packages/directory/addon/controllers/directory.js

```diff
@@ -33,6 +34,23 @@ export default Controller.extend({
    */
   q: '',
 
+  /**
+   * @property {String} title - Title for the table
+   */
+  title: computed('filter', 'sortBy', function() {
+    let title = dirInfo[0].name,
```

Comment:
How will we handle this when other directories are added?

Reply:
We will have to update the enum.  But we might need to change the route rules here slightly, when we know what they are.